src/whisper/types.ts

```diff
@@ -558,6 +543,28 @@ export type Autocomplete = SelectComponent<WhisperComponentType.Autocomplete> &
    * Default selected value
    */
   value?: string;
+} & (AutocompleteMatchSorter | AutocompleteFilter);
+
+export type AutocompleteFilter = {
+  /**
+   * Options to configure how the filter search behaves
+   *
+   * (Note: filterOptions and matchSorter can't be used together; if you include both,
+   * matchSorter will take precedence)
+   */
+  filterOptions?: AutocompleteFilterOptions;
+  matchSorter?: never;
+};
+export type AutocompleteMatchSorter = {
+  /**
+   * Options to use with our implementation of match-sorter
+   * https://github.com/kentcdodds/match-sorter
+   *
+   * (Note: filterOptions and matchSorter can't be used together; if you include both,
+   * matchSorter will take precedence)
+   */
+  matchSorter?: AutocompleteMatchSorterOptions;
+  filterOptions?: never;
```

Comment:
should filterOptions be type AutocompleteMatchSorterOptions instead of matchSorter?